packages/elements/src/internals/machines/sign-up/start.machine.ts

```diff
@@ -105,11 +115,20 @@ export const SignUpStartMachine = setup({
   }),
   entry: 'setDefaultFormValues',
   initial: 'Init',
+  on: {
+    SET_FORM: {
+      actions: 'setFormRef',
+    },
+  },
   states: {
     Init: {
       description:
         'Handle ticket, if present; Else, default to Pending state. Per tickets, `Attempting` makes a `signUp.create` request allowing for an incomplete sign up to contain progressively filled fields on the Start step.',
       always: [
+        {
+          guard: and(['hasTicket', 'isMissingRequirements']),
+          target: 'Pending',
+        },
```

Comment:
When the sign up attempt is missing requirements, users need to be able to submit the form, so we move to `Pending`